File: Client/src/dtos/authDtos.ts
export type RegisterDto = {
  UserName: string;
  Password: string;
};

export type LoginDto = {
  UserName: string;
  Password: string;
};

export type ResponseTokenDto = {
  accessToken: string;
  refreshToken: string;
  refreshTokenExpiresAt: string;
}

File: Server/src/Socialgram.Application/DTOs/AuthDtos.cs
namespace Socialgram.Application.DTOs
{
    public record class RegisterDto(string UserName, string Password);

    public record class LoginDto(string UserName, string Password);

    public record class ResponseTokenDto(string AccessToken,
                                         string RefreshToken,
                                         DateTime RefreshTokenExpiresAt);

    public record class RefreshTokenDto(string RefreshToken);
}
